index.js

```diff
@@ -240,6 +240,11 @@ class Session {
   options (opts) {
     this[kCookieOptions] = opts
   }
+
+  data () {
+    const { changed, deleted, ...data } = this[kObj]
+    return data
```

Comment:
[Q] I thought we would want to exclude `changed` and `deleted`, what do you think?

Reply:
I would keep them outside and, if needed, add a new getter:

`dataChanged() {}` and `dataDeleted()`

Reply:
If I am not wrong since we can `request.session.changed` and `request.session.deleted` there is no need for those two?

Reply:
me neither, could you add a test for it and check if it works as expected?